styleguide/components/21-atoms/inputs/input-checkbox/README.md

```diff
@@ -11,3 +11,5 @@ Use checkboxes
 * when several options from a list can be selected.
 * to select terms and conditions or options for another input field.
 * to opt-in/out.
+* when the user has 7 or less options to choose from. For higher numbers, use
```

Comment:
Than you can't select multiple options anymore :) Is multiselect used here, or chosen, ...

Reply:
Good question, I'll ask Johan